website/docs/quick-start-guide.md

```diff
@@ -125,25 +152,36 @@ pyspark \
 Hudi support using Spark SQL to write and read data with the **HoodieSparkSessionExtension** sql extension.
 From the extracted directory run Spark SQL with Hudi:
 
+```shell
+# Spark 3.4
+spark-sql --packages org.apache.hudi:hudi-spark3.4-bundle_2.12:0.14.0 \
+--conf 'spark.serializer=org.apache.spark.serializer.KryoSerializer' \
+--conf 'spark.sql.extensions=org.apache.spark.sql.hudi.HoodieSparkSessionExtension' \
+--conf 'spark.sql.catalog.spark_catalog=org.apache.spark.sql.hudi.catalog.HoodieCatalog' \
+--conf 'spark.kryo.registrator=org.apache.spark.HoodieSparkKryoRegistrar'
+```
 ```shell
 # Spark 3.3
 spark-sql --packages org.apache.hudi:hudi-spark3.3-bundle_2.12:0.13.0 \
 --conf 'spark.serializer=org.apache.spark.serializer.KryoSerializer' \
 --conf 'spark.sql.extensions=org.apache.spark.sql.hudi.HoodieSparkSessionExtension' \
---conf 'spark.sql.catalog.spark_catalog=org.apache.spark.sql.hudi.catalog.HoodieCatalog'
+--conf 'spark.sql.catalog.spark_catalog=org.apache.spark.sql.hudi.catalog.HoodieCatalog' \
```

Comment:
how come our default quick start is referring to 0.13.0 instead of 0.13.1 ?

Reply:
should fix it to 0.13.1.

Reply:
0.13.1 docs are independently revised.

Reply:
Fixed to 0.14.0